### DISCUSSION TITLE
The Status page always shows client offline when connecting to broker

### DISCUSSION THREAD

**Discussion by win5923** [vernemq/vernemq#2326] [General]
When I connect using JMeter or MQTT Explorer, the client always shows as offline, regardless of whether there is any pub/sub activity. Anyone know what happened?

**Comment by ioolkos** [comment]
@win5923 The status page should update every 10  seconds or so. Also, make sure to not have races due to using the same ClientID. A quick look into the logs might help you as well.

**Reply by win5923** [reply]
hi, @ioolkos, thanks for your reply. 
The logs do not output anything, and when I connect using MQTT Explorer, it shows offline.
![image](https://github.com/vernemq/vernemq/assets/56353753/37bbb55f-2cdf-4f88-9205-2d4f13b0a114)

**Comment by ioolkos** [comment]
`vmq-admin session show --help`

**Reply by win5923** [reply]
Looks like the status page is not displaying the correct information.
![image](https://github.com/vernemq/vernemq/assets/56353753/477a49ed-daac-4f94-9171-b0c7b5961f27)
